tsconfig.json

```diff
@@ -10,6 +10,7 @@
   },
   "include": [
     "./node_modules/wix-ui-core/dist/src/globals.d.ts",
+    "./node_modules/storybook-snapper/src/external-types.d.ts",
```

Comment:
why is this required?

Reply:
Good question.
I was having some problems with importing the library.
It might not be needed. I'll check.